samples/nrf5340/netboot/src/main.c

```diff
@@ -39,14 +39,13 @@ int main(void)
 
 	uint32_t s0_addr = s0_address_read();
 	bool valid = false;
-	uint8_t status = pcd_fw_copy_status_get();
-
+	uint8_t status = pcd_cmd_status_get();
 #ifdef CONFIG_PCD_READ_NETCORE_APP_VERSION
 	if (status == PCD_STATUS_READ_VERSION) {
 		err = pcd_find_fw_version();
 		if (err < 0) {
 			printk("Unable to find valid firmware version %d\n\r", err);
-			pcd_fw_copy_invalidate();
+			goto failure;
```

Comment:
I don't want it to call `pcd_done`. I know this changes the behaviour a bit but calling done here would indicate that the command completed successfully which is not true. Meaning that if you read out the data at this point it will just be garbage.

Reply:
Thinking about this a bit more you are correct and it should invalidate and proceede